src/package/nuspec/Microsoft.NET.Test.Sdk.props

```diff
@@ -1,20 +1,27 @@
 <!--
-***********************************************************************************************
-Microsoft.NET.Test.Sdk.props
-
-WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
-          created a backup copy.  Incorrect changes to this file will make it
-          impossible to load or build your projects from the command-line or the IDE.
-
-Copyright (c) .NET Foundation. All rights reserved. 
-***********************************************************************************************
+ ***********************************************************************************************
+ Microsoft.NET.Test.Sdk.props
+ 
+ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+           created a backup copy.  Incorrect changes to this file will make it
+           impossible to load or build your projects from the command-line or the IDE.
+ 
+ Copyright (c) .NET Foundation. All rights reserved. 
+ ***********************************************************************************************
 -->
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
This was earlier conditioned only for .netcore scenario , I'm not sure what it does, but do we need it for netfx?